### PR TITLE
feat: Initial implementation of Gladia transcriptions to BBB 3.0 (html5)

### PR DESCRIPTION
**Pull request by JoVictorNunes**
### What does this PR do?

Following up #19841, this PR is the first attempt to implement the frontend portion for Gladia transcriptions on 3.0. It depends on #19841 being migrated to GraphQL.

### Closes Issue(s)

Closes none

### CODE REVIEW

**Comment by JoVictorNunes** [comment]
This PR is ready to be reviewed.

File: bigbluebutton-html5/public/locales/en.json [hunk 1]
```diff
@@ -624,6 +628,8 @@
     "app.settings.dataSavingTab.webcam": "Enable other participants webcams",
     "app.settings.dataSavingTab.screenShare": "Enable other participants desktop sharing",
     "app.settings.dataSavingTab.description": "To save your bandwidth adjust what's currently being displayed.",
+    "app.settings.transcriptionTab.partialUtterances": "Show partial utterances",
```
Comment (antobinary) [inline bigbluebutton-html5/public/locales/en.json:631]: @lfzawacki Have you considered renaming the labels to be more descriptive? For example: "Display the interim transcription (when disabled we will only display results that have been refined by analyzing the full sentence)"

File: bigbluebutton-html5/public/locales/en.json [hunk 2]
```diff
@@ -624,6 +628,8 @@
...
+    "app.settings.transcriptionTab.minUtteranceLength": "Minimum utterances length (seconds)",
```
Comment (antobinary) [inline bigbluebutton-html5/public/locales/en.json:632]: Have you considered renaming the labels to be more descriptive? For example: "Length of the speech (in seconds) used for transcription. Longer portion typically transcribe better but also result in longer delay before displaying"

File: bigbluebutton-html5/private/config/settings.yml [hunk 3]
```diff
@@ -240,6 +241,12 @@ public:
       dataSaving:
         viewParticipantsWebcams: true
         viewScreenshare: true
+      # Options that are sent to the transcription backed (only Gladia is supported for now)
+      transcription:
+        # Indicates if the transcription backend should include partial results
+        partialUtterances: true
+        # The minumum length (in seconds) an utterance has to have for we to use it
+        minUtteranceLength: 1
```
Comment (antobinary) [inline bigbluebutton-html5/private/config/settings.yml:249]: Note: Setting it to `3` for Bulgarian improved the quality of the transcription significantly. I also combined it with disabling partialUtterances though.

File: bigbluebutton-html5/imports/api/users-settings/server/methods/addUserSettings.js [hunk 4]
```diff
@@ -67,6 +67,10 @@ const currentParameters = [
   'bbb_change_layout',
   'bbb_direct_leave_button',
   'bbb_default_layout',
+  // TRANSCRIPTION
+  'bbb_transcription_partial_utterances',
```
Comment (antobinary) [inline bigbluebutton-html5/imports/api/users-settings/server/methods/addUserSettings.js:71]: TODO: we need to document these in https://github.com/bigbluebutton/bigbluebutton/blob/v2.7.x-release/docs/docs/administration/customize.md?plain=1#L1523

**Comment by JoVictorNunes** [comment]
Closing in favor of https://github.com/bigbluebutton/bigbluebutton/pull/20199.

**Pull request closed without merging**